content/news/XXXX-XX-XX-mixxx-2-4-1-released.md

```diff
@@ -0,0 +1,151 @@
+  [#10481](https://github.com/mixxxdj/mixxx/issues/10481)
+* Require a minimum movement before initiating the drag&drop of tracks
+  [#13135](https://github.com/mixxxdj/mixxx/pull/13135)
+  [#12902](https://github.com/mixxxdj/mixxx/issues/12902)
+* iTunes/Serato/Traktor/Rhythmbox: Print error if library file could not be opened
+  [#13012](https://github.com/mixxxdj/mixxx/pull/13012)
+* Playlists: improve table update after deleting (purging) track files
+  [#13127](https://github.com/mixxxdj/mixxx/pull/13127)
+* Fix Color column width issue
+  [#12852](https://github.com/mixxxdj/mixxx/pull/12852)
+* Tracks: select track row when clicking the preview button (only when starting preview)
+  [#12791](https://github.com/mixxxdj/mixxx/pull/12791)
+* Library track menu: show Hide action also in Playlist & Crates
+  [#11901](https://github.com/mixxxdj/mixxx/pull/11901)
+
+#####Miscellaneous
+* AutoDJ: Fix button state after error message about playing deck 3/4
+  [#12976](https://github.com/mixxxdj/mixxx/pull/12976)
+  [#12975](https://github.com/mixxxdj/mixxx/issues/12975)
+* Tagfetcher: Cache fetched covers
+  [#12301](https://github.com/mixxxdj/mixxx/pull/12301)
+  [#11084](https://github.com/mixxxdj/mixxx/issues/11084)
+* Tagfetcher: Cache fetched covers
+  [#12301](https://github.com/mixxxdj/mixxx/pull/12301)
+  [#11084](https://github.com/mixxxdj/mixxx/issues/11084)
+* Avoid beats iterator being one off and DEBUG_ASSERT in Beats::iteratorFrom
+  [#13150](https://github.com/mixxxdj/mixxx/pull/13150)
+  [#13149](https://github.com/mixxxdj/mixxx/issues/13149)
+* Show hint if resource path in CMakeCache.txt does not exist
+  [#12929](https://github.com/mixxxdj/mixxx/pull/12929)
+* Always calculate the auto value for colorful console output.
+  [#13153](https://github.com/mixxxdj/mixxx/pull/13153)
+* Fix FLAC recording on macOS and Windows
+  [#10880](https://github.com/mixxxdj/mixxx/issues/10880)
+  [#13154](https://github.com/mixxxdj/mixxx/pull/13154)
+
+
+####What happened?
+I would like to share some memorable moments I could follow in the [Zulip](https://mixxx.zulipchat.com/login/)-chat that are significant for the atmosphere and
+motivation of the Mixxx-team:
+
+#####Not giving up...
+Ten days ago a Mixxx-er posted a problem that he struggled with for over a year.
+He was unable to make a MIDI-connection between his Pioneer DJM-750MK2 Mixer with Mixxx and then activate the DVS (Digital Vinyl System) to work with Mixxx's Vinyl Control.
+A Mixxx-developer took the challenge and during 2 weeks you could see the messages flying over: log-files, screenshots, debugging-info, tooltests, capturing communication, a new pull request...
+I think I've seen over 100 messages.
+Trying things, fail, one step forward, two steps back ...
+After two weeks the Mixxx-developer managed to get it fixed.
+All on distance, not having the hardware available, putting a lot of time in it.
+It was fantastic to be a witness of this magic.
+######Shout out to N/S!
```

Comment:
Thanks for the shoutout. Feel free to link my author page https://mixxx.org/news/author/nikolaus-einhauser/

Regardless, I'm a little skeptical of this segment because this is teasing things that are nowhere near finished right now.

Reply:
you're welcome,
It's about the motivation. The atmosphere, what you did to help Phr3d13.
What I've seen was .... as I described.
You should  be more proud, all of you. 
This is not just something that can be taken for granted.

Reply:
I regret but I understand, I don't want you to feel unconfortable about it.
A last try: not even when I write about 'a piece of hardware" and  without your name under it so it becomes quiet anonymous for 99,9% of the readers?

Reply:
sure. I still personally don't think its very relevant for most readers, but you're the author so I'll let you decide.